packages/aws-lambda-otel-extension/opt/otel-extension/helper.js

```diff
@@ -107,8 +107,8 @@ const resourceAttributes = [
   },
   {
     key: 'faas.collector_version',
-    value: '@serverless/aws-lambda-otel-extension-0.1.8',
-    source: '@serverless/aws-lambda-otel-extension-0.1.8',
+    value: '@serverless/aws-lambda-otel-extension-0.1.9',
+    source: '@serverless/aws-lambda-otel-extension-0.1.9',
```

Comment:
This ideally should be read automatically, or generated automatically for publication.

I'll solve when working on layer improvements